### ISSUE TITLE
What prevents network-sidechannel eavesdropping deanonymization?

### ISSUE THREAD

**Issue by FishmanL** [DP-3T/documents#189]
In particular. given that uninfected people are making data-light get requests, and infected people are making data-heavy post requests (with all the ephemeral identifiers) what's stopping an attacker from snooping all encrypted traffic and tracking data volume of each transmission to match infected people with IP addresses?

(There seems a simple solution, which is to have uninfected people every so often send in lists of ephemeral identifiers equal to some random subset of identifiers they have in local storage as 'at risk')

**Comment by lbarman** [comment]
Hi @FishmanL, thanks for the input! You're right, clients should also do dummy uploads in this case. This will be precised in the next version of the whitepaper.
> (There seems a simple solution, which is to have uninfected people every so often send in lists of ephemeral identifiers equal to some random subset of identifiers they have in local storage as 'at risk')

Another option would be to simply upload a random seed instead of SK_t, which seems fairly equivalent.
Thanks!

**Comment by marado** [comment]
If "this will be precised in the next version of the whitepaper", shouldn't the issue remain open until such version is published?

**Comment by FishmanL** [comment]
Also, dummy data doesn't quite fix the problem -- if you consistently request ID lists from the server, you can tell whether someone's upload is dummy data or real data.  You need to batch uploads + allow dummy uploads to fix this.

**Comment by lbarman** [comment]
@marado I commented on the internal whitepaper, it's waiting for someone to include the comments and commit to git. We have to somehow close the issue as we get a constant stream of them; to me this one was addressed on our side :) if you have something to add to the discussion you're welcome.

@FishmanL 
> if you consistently request ID lists from the server, you can tell whether someone's upload is dummy data or real data. 

Very good point! it seems that batching *downloads* is easier to do (e.g., the server produces a new version everyday). It also facilitate intermediate caching if the data doesn't change constantly. Adding this "differential attack" note on the dummy upload section.
